ocaml/xapi/license_check.ml

```diff
@@ -13,27 +13,34 @@
  *)
 module L = Debug.Make (struct let name = "license" end)
 
-let never, _ =
-  let start_of_epoch = Unix.gmtime 0. in
-  Unix.mktime {start_of_epoch with Unix.tm_year= 130}
+module Date = Xapi_stdext_date.Date
+
+let never = Ptime.of_year 2100 |> Option.get |> Date.of_ptime
+
+let serialize_expiry = function
+  | None ->
+      "never"
+  | Some date when Date.equal date never ->
```

Comment:
Should we test for `date >= never`?

Reply:
technically, only that timestamp is supposed to be never